pkg/runtime/runtime.go

```diff
@@ -305,6 +311,36 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 	return nil
 }
 
+func (a *DaprRuntime) populateSecretsConfiguration() {
+	for name := range a.secretStores {
+		// This is added for the scenario where secret store component is added, but
+		// no restrictive configuration is defined.
+		a.secretsConfiguration[name] = &config.ParsedSecretsConfiguration{
+			DefaultAccess: config.AllowAccess,
+		}
+		// a.defaultSecretAccess[name] = config.AllowAccess
+		for _, storeInList := range a.globalConfig.Spec.Secrets.Scopes {
+			if storeInList.StoreName == name {
+				c := a.secretsConfiguration[name]
+				c.ParseSecretsDefaultAccess(storeInList.DefaultAccess)
+				// if deny, break. Eg: kubernetes is put as deny access with no allowed list.
+				// This scenario also ignores only deniedSecrets being populated if the defaultAccess is deny.
```

Comment:
This contradicts with https://github.com/dapr/dapr/pull/2060/files#r490600930, since if the first store in the list causes a denial, it will takes precedence?

Reply:
good catch ... will fix it.

Reply:
good catch.